STYLE-GUIDE.md

```diff
@@ -0,0 +1,56 @@
+# Style Guide
+
+This document defines some general styles we adhere to in the docs.
+
+## Words are important
+
+- Instead of _guys_ try _folks_, _yall_, or _everyone_
+- Instead of _crazy_ try _wild_
+- Instead of _click_ use _select_
+- Instead of _dummy_ use _placeholder_
+- Avoid using gendered language
+- Avoid using violent language (e.g., _kill_)
```

Comment:
Do we want to add an alias to the `pulumi destroy` CLI command for people that might want to use less violent language?